worker/appm/store/store.go

```diff
@@ -1183,6 +1183,10 @@ func (a *appRuntimeStore) evtEventHandler() cache.ResourceEventHandlerFuncs {
 	return cache.ResourceEventHandlerFuncs{
 		AddFunc: func(obj interface{}) {
 			evt := obj.(*corev1.Event)
+			if evt.Reason != "SuccessfulRescale" {
+				return
```

Comment:
It is recommended to print the log.

Reply:
UpdateFunc will be triggered continuously, and there will be many logs. You can use kubectl to view hpa events.